packages/celotool/geth_tests/src/lib/utils.ts

```diff
@@ -433,9 +440,14 @@ export function getHooks(gethConfig: GethTestConfig) {
     }
   }
 
+  const restartInstance = async (instance: GethInstanceConfig) => {
```

Comment:
I wouldn't return this in `getHooks`, since those are intended for `before`, `after`, etc. hooks.  Since this is only used in one place, consider just calling `killInstance` and `initAndStartGeth` directly in the test, and adding a comment to make it clear that this clears the data directory

Reply:
I'm going to keep this in hooks since i've update some of the other tests to use this restart instance in the before hooks